libs/core/kiln_ai/adapters/ml_model_list.py

```diff
@@ -1239,6 +1241,25 @@ class KilnModel(BaseModel):
             ),
         ],
     ),
+    # Claude Opus 4.5
+    KilnModel(
+        family=ModelFamily.claude,
+        name=ModelName.claude_opus_4_5,
+        friendly_name="Claude Opus 4.5",
+        providers=[
+            KilnModelProvider(
+                name=ModelProviderName.openrouter,
+                model_id="anthropic/claude-opus-4.5",
+                structured_output_mode=StructuredOutputMode.function_calling,
+            ),
+            KilnModelProvider(
+                name=ModelProviderName.anthropic,
+                model_id="claude-opus-4-5-20251101",
+                structured_output_mode=StructuredOutputMode.function_calling,
```

Comment:
should be json_schema. Merge main if you had issues before, but it should work with latest main.